backend/db/sql/schema.sql

```diff
@@ -125,14 +124,12 @@ CREATE TABLE IF NOT EXISTS EOS_parent_survey
 
 CREATE TABLE IF NOT EXISTS Semester_tutoring_obj
 (
-    semester_tutoring_id BINARY(16) DEFAULT (UUID_TO_BIN(UUID(), 1)) PRIMARY KEY,
+    semester_tutoring_obj_id BINARY(16) DEFAULT (UUID_TO_BIN(UUID(), 1)) PRIMARY KEY,
```

Comment:
before we commit this we would need to check all of our dao stuff that uses prepared statements and change it to the new key name

Reply:
would it make more sense to just revert my stuff to make sure it doesn't break everyone else's? lol